<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageFFLives</class>
 <widget class="QWidget" name="PageFFLives">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Elite Starting Lives</string>
     </property>
     <property name="buddy">
      <cstring>fieldStartingLivesElite</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="fieldMaxExtraLivesSpartan">
     <property name="toolTip">
      <string>Control the maximum number of lives that the defending team is allowed to have.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>unlimited</string>
     </property>
     <property name="suffix">
      <string> lives</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>126</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="fieldStartingLivesElite">
     <property name="toolTip">
      <string>Control the number of lives that players on the attacking team start with.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>unlimited</string>
     </property>
     <property name="suffix">
      <string> lives</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>126</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="fieldStartingLivesSpartan">
     <property name="toolTip">
      <string>Control the number of lives that the defending team starts with.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>unlimited</string>
     </property>
     <property name="suffix">
      <string> lives</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>126</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Spartan Starting Lives</string>
     </property>
     <property name="buddy">
      <cstring>fieldStartingLivesSpartan</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Reward for Killing Elite Players</string>
     </property>
     <property name="buddy">
      <cstring>fieldEliteKillBonus</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Max Spartan Extra Lives</string>
     </property>
     <property name="buddy">
      <cstring>fieldMaxExtraLivesSpartan</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="fieldEliteKillBonus">
     <property name="toolTip">
      <string>Control how many extra lives the defending team gains for killing an Elite player.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> lives</string>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fieldStartingLivesSpartan</tabstop>
  <tabstop>fieldMaxExtraLivesSpartan</tabstop>
  <tabstop>fieldEliteKillBonus</tabstop>
  <tabstop>fieldStartingLivesElite</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
